tests/unit/commands/local/cli_common/test_invoke_context.py

```diff
@@ -1082,5 +1082,8 @@ def test_must_pass_custom_region(self, get_stacks_mock):
         invoke_context = InvokeContext("template_file", aws_region="my-custom-region")
         invoke_context._get_stacks()
         get_stacks_mock.assert_called_with(
-            "template_file", parameter_overrides=None, global_parameter_overrides={"AWS::Region": "my-custom-region"}
+            "template_file",
+            parameter_overrides=None,
+            root_template_dir=os.getcwd(),
+            global_parameter_overrides={"AWS::Region": "my-custom-region"},
```

Comment:
Can you put a comment at line 1087 explaining a little bit why it is `os.getcwd()`?

Reply:
Sure!